docs/en-US/component/rate.md

```diff
@@ -83,32 +83,43 @@ Use `css/scss` language to change the global or local color. We set some global
+| icons                 | icon components. If array, it should have 3 elements, each of which corresponds with a score level, else if object, the key should be threshold value between two levels, and the value should be corresponding icon component | ^[object]`string[] \| Component[] \| Record<number, string \| Component>` | [StarFilled, StarFilled, StarFilled]                               |
+| void-icon             | component of unselected icons                                                                                                                                                                                                  | ^[string] / ^[Component]                                                  | Star                                                               |
+| disabled-void-icon    | component of unselected read-only icons                                                                                                                                                                                        | ^[string] / ^[Component]                                                  | StarFilled                                                         |
+| show-text             | whether to display texts                                                                                                                                                                                                       | ^[boolean]                                                                | false                                                              |
+| show-score            | whether to display current score. show-score and show-text cannot be true at the same time                                                                                                                                     | ^[boolean]                                                                | false                                                              |
+| text-color            | color of texts                                                                                                                                                                                                                 | ^[string]                                                                 | #1F2D3D                                                            |
+| texts                 | text array                                                                                                                                                                                                                     | ^[string&#91;&#93;]                                                       | ['Extremely bad', 'Disappointed', 'Fair', 'Satisfied', 'Surprise'] |
```

Comment:
Yes, because there is only one array type, I want to use `^[string[]]`, but the current md analysis does not support it, so I can only use escape characters.

Reply:
Because it was not considered in the design

^[JavaScript type]\`TypeScript type\`

Reply:
done.